### ISSUE TITLE
Add a Filter for a person like the filter on the kanban board

### ISSUE THREAD

**Issue by paule96** [microsoft/vsts-extension-retrospectives#10]
Hi,

is it possible to implement a filter for persons on every page of this extension? So you can on the feedback of a specific person.

Like this:

![image](https://user-images.githubusercontent.com/11291885/81073570-1be87f00-8ee8-11ea-8f31-4021e1771f53.png)

**Comment by ArieHein** [comment]
Interesting to understand the use case. How many people are in the team and how many similar items do they enter each sprint?  Ideally you would group the ones that are similar to make it easy to later vote. In a way filtering by people actually removes visibility of similar input from others.

**Comment by paule96** [comment]
Yes but sometimes your meeting is a little bit different. Sometimes we want to talk about the feedback of each person and not of each type. So we first listen sometimes to all of the feedback of one person and switch to the next person. 

Most of the time we do it per type, but sometimes its good to do it for each person.

In the specific team are ~10 people. so not too big. 

> By the way, I like the grouping feature. That's a very nice feature.
